### ISSUE TITLE
[1.16.5] Recipes not working

### ISSUE THREAD

**Issue by ghost** [TwistedGate/ImmersivePosts#35]
### Description of the issue
Posts recipes doesn't load since 4.2.0. 
Can't craft it or see it on JEI. Can replicate it with a bare bones instance. 
Noticed this error repeated for each material:
[Render thread/ERROR] [immersiveposts/Config/]: [IPOConfigCondition]: Unknown key "uranium"
Tried deleting all configs and it didn't solve it.

### Versions and Modlist
ImmersiveEngineering-1.16.5-5.0.3-138
immersiveposts-1.16.5-4.2.1 [saw the same error with with 4.2.0]
jei-1.16.5-7.7.1.121

On Forge 36.2.4 

### Crash Log
It doesn't crash.

**Comment by TwistedGate** [comment]
Can't reproduce it, neither with released version or in dev. Not sure what is up with that.

As for the `Unknown key "uranium" Tried deleting all configs and it didn't solve it.`
Unless you're going to change the config settings that can be safely ignored, havent actualy figured out yet why that happens.

**Comment by TwistedGate** [comment]
c5aae30cebe5d8098401978d6a94274796c9bacb

**Comment by ghost** [comment]
Well, given you can't reproduce and the tests I made [checked different javas too]. Then it should be my system (Debian Sid, using MultiMC), that is the only thing I can think of, an issue somewhere in my OS libraries, who knows where tho. Idk what would be causing that, but I didn't see anything similar happening with other mods so far, and even yours before 4.2.0 worked fine. 
I will correct myself tho, it isn't only the fences but also the rods recipe that are conditional [on the crafting table], it loading the recipes from IE made me assume the rods were loading fine.
So basically all 17 conditioned recipes fails on my instances, and that error Unknown key X fires up 17 times. 
If it comes down to it tho, I will just remove the condition through a datapack, so that the recipes I want load and then remove any I don't want on KubejJS.

**Comment by TwistedGate** [comment]
Should hopefully be fixed with `4.2.2` now
